tests/commands/test_account.py

```diff
@@ -320,7 +349,10 @@ async def test_send_defaults(
 
 
 @pytest.mark.asyncio
-@patch("nile.core.account.deploy_account", return_value=(MOCK_ADDRESS, MOCK_INDEX))
+@patch(
+    "nile.core.account.deploy_account",
+    return_value=(MOCK_ADDRESS, MOCK_TX_HASH, MOCK_ABI),
```

Comment:
it looks like we could abstract the whole tuple into a variable, to simplify these changes in the future

Reply:
Done!